### PR TITLE
kernel-uki: remove noxsaves parameter from cmdline

### PR DESCRIPTION
**Pull request by trungams**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./LICENSES-AND-NOTICES/SPECS/data/licenses.json`, `./LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md`, `./LICENSES-AND-NOTICES/SPECS/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Removing noxsaves from UKI cmdline since Hyper-V fix has been rolled out on Azure.

###### Change Log  

- Remove noxsaves parameter from UKI cmdline

###### Does this affect the toolchain?  

**YES**

###### Associated issues  

- https://microsoft.visualstudio.com/OS/_workitems/edit/51618524/

###### Test Methodology

- Local build succeeds
- Built a CVM image and deployed on Azure. Boot succeeds

### CODE REVIEW

**Review by Camelron — APPROVED** [review]

File: SPECS/kernel/kernel-uki.spec [hunk 1]
```diff
@@ -13,12 +13,12 @@
 # We need to temporarily turn it off by disabling xsaves until the problem
 # is fixed on Azure. Since shadow stack depends on xsaves, disabling xsaves
 # ensures the feature bit for shadow stack is also turned off.
-%define cmdline console=ttyS0 noxsaves
+%define cmdline console=ttyS0
```
Comment (Camelron) [inline SPECS/kernel/kernel-uki.spec:16]: Is this really all we need for kernel-uki commandline?

Comment (trungams) [reply SPECS/kernel/kernel-uki.spec:16]: We might want to add more to make it on par with the regular marketplace image, but this is all we need to make the image work for now

Comment (Camelron) [reply SPECS/kernel/kernel-uki.spec:16]: changes lgtm

**Review by ms-mahuber — APPROVED** [review]

File: SPECS/kernel/kernel-uki.spec [hunk 2]
```diff
@@ -13,12 +13,12 @@
 # We need to temporarily turn it off by disabling xsaves until the problem
```
Comment (reubeno) [inline SPECS/kernel/kernel-uki.spec:13]: Is this comment now outdated? Should we remove it?

Comment (trungams) [reply SPECS/kernel/kernel-uki.spec]: True. Thank you for the catch

**Review by christopherco — APPROVED** [review]

**Pull request merged**